stock_pack_operation_auto_fill/models/stock_move.py

```diff
@@ -8,12 +8,25 @@
 class StockMove(models.Model):
     _inherit = 'stock.move'
 
+    def _create_extra_move(self):
+        """
+        When user set on stock move line done a quantity greater than initial
+        demand Odoo creates an extra stock move with the difference and it is
+        posible to create an extra stock move line with qty_done = 0 which will
+        be deleted in _action_done method.
+        This method set a context variable to prevent set qty_done for these
+        cases.
+        """
+        my_self = self.with_context(skip_auto_fill_operation=True)
```

Comment:
You should change context if move.picking_id.auto_fill_operation is True. Without that, you will create new environnment at each extra move creation.